styles/global.scss

```diff
@@ -190,3 +190,8 @@ body {
 hr {
   background: $k-grey;
 }
+
+.icon svg {
+  width: 1rem;
+  height: 1rem;
```

Comment:
Why we would like to set a fixed size for every icon 🤔 If so, we have to override the icon size in many places.

Reply:
Alright. Maybe setting the default size to 1rem would work for most of the icons : ) 

<img width="1400" alt="image" src="https://user-images.githubusercontent.com/31397967/215273581-40ed94eb-2e00-4eb0-8308-6da194d7c69e.png">

Let's bring back this style.
```
.icon svg {
  width: 1rem;
  height: 1rem;
}
```

Reply:
I hope so, however, It does not only appear on explore pages but also on other pages : )

<img width="331" alt="image" src="https://user-images.githubusercontent.com/31397967/215276920-958da4fa-2c68-4731-9cd7-5f82771c4a4c.png">

<img width="803" alt="image" src="https://user-images.githubusercontent.com/31397967/215276932-77bd59a7-d847-4fe6-9d48-d26b61ea728e.png">